### PR TITLE
EZP-30681: Fixed double slash in path_identification_string after moveSubtree

### PR DESCRIPTION
**Pull request by SerheyDolgushev**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-30681](https://jira.ez.no/browse/EZP-30681)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `master`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

Use case:
1. Under "Home" location (#2) create two folders: "Parent" and "Move me".
2. Move "Move me" folder to "Parent".
3. Check `ezcontentobject_tree.path_identification_string` for moved node.

Expected result: `node_2/parent/move_me`
Actual result: `node_2/parent//move_me`

**TODO**:
- [X] Implement feature / fix a bug.
- [X] Implement tests.
- [X] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [X] Ask for Code Review.

### CODE REVIEW

**Comment by SerheyDolgushev** [comment]
Please note, existing install might require to run SQL to replace `//` => `/` in `ezcontentobject_tree.path_identification_string`

**Comment by SerheyDolgushev** [comment]
@alongosz, please have a look

**Comment by SerheyDolgushev** [comment]
@alongosz 
1. Rebuild is done
2. The issue is fixed. Please note `testMoveHiddenSourceChildUpdate` had en edge case: moving location with root `path_identification_string`. So current fix handles cases like that one.
3. The test is added.

Please review.

File: eZ/Publish/API/Repository/Tests/LocationServiceTest.php [hunk 1]
```diff
@@ -2088,6 +2088,54 @@ public function testMoveSubtreeThrowsInvalidArgumentException()
...
+            $this
+                ->createFolder(['eng-US' => 'Move Me'], $topNode->contentInfo->mainLocationId)
+                ->contentInfo
+                ->mainLocationId
+        );
+
+        $locationService->moveSubtree($location, $newParentLocation);
+
+        // path location string is not present on API level, so we need to query database
+        $connection = $this->getSetupFactory()->getServiceContainer()->get('ezpublish.api.storage_engine.legacy.dbhandler')->getConnection();
```
Comment (mikadamczyk) [inline eZ/Publish/API/Repository/Tests/LocationServiceTest.php]: Can we divide it into several lines? It is too long

**Comment by alongosz** [comment]
@SerheyDolgushev in the future please fetch latest upstream changes before doing rebase. You've rebased against 6.7 from September 2019, which gave our QA a headache :stuck_out_tongue_winking_eye:

This time I fixed that for you and force-pushed :) // cc @barbaragr

**Comment by lserwatka** [comment]
You can merge it up

**Comment by alongosz** [comment]
Done

**Comment by SerheyDolgushev** [comment]
@alongosz should a new item be added to PR todo list? Something like "Check that you are using the latest source branch"?

**Comment by alongosz** [comment]
> @alongosz should a new item be added to PR todo list? Something like "Check that you are using the latest source branch"?

Maybe, but I wonder if this would be understood out of context. Moreover some people (not gonna point fingers ;p) tend to check boxes regardless of if they did the thing, so I started to find this pointless :P 
At some point I planned to revise PR template anyway.
